user_guide_src/source/database/configuration/011.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Config;
+
+use CodeIgniter\Database\Config;
+
+class Database extends Config
+{
+    // ...
+
+    // MySQLi over a socket
+    public array $default = [
+        // ...
+        'hostname' => '/cloudsql/toolbox-tests:europe-north1:toolbox-db"',
```

Comment:
`"` should be removed?

```suggestion
        'hostname' => '/cloudsql/toolbox-tests:europe-north1:toolbox-db',
```

Reply:
All your changes look good. Sorry for missing that quote.